tgui/packages/tgui/interfaces/RCD.js

```diff
@@ -95,6 +95,11 @@ const AirlockSettings = (props, context) => {
             icon="pen-alt"
             content="Edit"
             onClick={() => modalOpen(context, 'renameAirlock')}
+          /><Button
+            icon={electrochromic ? 'toggle-on' : 'toggle-off'}
+            content="Electrochromic"
+            selected={electrochromic}
+            onClick={() => act(electrochromic ? 'electrochromic_off' : 'electrochromic_on')}
```

Comment:
Can any door be electrochromic?

Reply:
Just glass ones. Toggling it on but not selecting a glass type will just not make it polarized when constructing the airlock.

Reply:
in that case, can you make it so the button only displays (or is only enabled) when the selected airlock is a glass type?